source/includes/api/_dataset.md

```diff
@@ -859,6 +859,8 @@ For more details on the characteristics of each of the basic data types, refer t
 
 ## Uploading a Dataset (Binary)
 
+To upload a dataset, you will need an authorization token of an user with role of **MANAGER** or greater. Follow the steps of [this guide to get your token](#generate-your-own-oauth-token) or [check here if you are having problems related with authentication](#troubleshooting-auth-problems).
```

Comment:
The rules are not so simple, check the code and update the PR accordingly

Reply:
Applied the same changes here as in the "Create a dataset" section (i.e. removed references to role hierarchy and updated text to mention it is only required a valid token for uploading a dataset).

From what I understood from the source code, the middlewares that might fail the access to uploading a dataset are due to **validation** _(in which case a 400 Bad Request is returned, so non-related with authentication)_ or **authorization** _(only a token is required, as with creating a dataset, no specific roles required)_.

Are there any rules besides these that I have missed?